content/applications/finance/payment_acquirers/adyen.rst

```diff
@@ -2,14 +2,15 @@
 Adyen
 =====
 
-`Adyen <https://www.adyen.com/>`_ is a Dutch-based company that offers several online payment
+`Adyen <https://www.adyen.com/>`_ is a Dutch company that offers several online payment
 possibilities.
 
-Configuration
-=============
-
 .. seealso::
    - :ref:`payment_acquirers/add_new`
+   - :doc:`../payment_acquirers`
+
+Configuration
+=============
```

Comment:
2 things are bothering me here:

1) The `add_new` anchor is about the configuration, so why not keep it there?
2) Is there a reason to take off this `See also` from the end and put it above? 

But then, if it was decided to do this for all acquirers, this is fine I guess, but since it'd only be the case for Adyen, that seems strange though.

Reply:
I understand the logic better now. Indeed, it makes more sense. I changed it back down.